### DISCUSSION TITLE
What should be used for real-time messaging within the project?

### DISCUSSION THREAD

**Discussion by Enet4** [Enet4/dicom-rs#264] [Ideas]
I had Gitter set up for a while, but there are multiple issues with the software, and I never get notified of any message which comes up. As such, I am looking for alternatives.

There are not many requirements that I can think of. It should be public to all contributors, easily shareable, and have a minimally decent experience on mobile devices.

- Discord?
- Zulip?
- ...others?

**Comment by troelsarvin** [comment]
I think Zulip is a good bet. I've seen it being used quite extensively in an HL7 related project I sometimes watch. Zulip is supposed to have a mobile device option, but I haven't tried that.

**Reply by Enet4** [reply]
Thank you for your feedback. I have decided to move the project's chat platform to Zulip: https://dicom-rs.zulipchat.com (see also #271)
